src/services/ui/src/pages/dashboard/Lists/spas/consts.tsx

```diff
@@ -44,7 +66,7 @@ export const useSpaTableColumns = (): OsTableColumn[] => {
         props?.isCms &&
         !(props.user?.["custom:cms-roles"] === UserRoles.HELPDESK)
           ? data.cmsStatus
-          : data.stateStatus,
+          : getStateStatusWithSubStatus(data),
```

Comment:
Based on our current display in `master`, this seems like a semantic error.

```typescript
{user?.isCms && checker.isInSecondClock && (
    <span id="secondclock">2nd Clock</span>
)}
```
This, to me, indicates 2nd clock is a CMS sub-status that should display for CMS users on their dashboard.

Reply:
Our current sub-statuses are "RAI Response Enabled/Disabled" for state, and "2nd clock" for CMS, according to the conditions found in the Details page that currently display the values. One of them is copied above for example. That says "if a user is CMS and in second clock, show the 2nd Clock sub-status". It stands to reason that if we are showing sub-statuses on the dashboard, we should be showing both the state AND cms sub statuses to the proper users, yes? This set up currently only shows the sub-status to state users.

Reply:
The ticket seems loose on requirements. Hunt down the answer to whether or not this is the expected behavior, but if we're adding a defacto "sub status" value to the dashboard for state and cms like the AC indicates, it shouldn't _just_ be the Enable/Disable substatus in my opinion.